Classes/Domain/Repository/DocumentRepository.php

```diff
@@ -789,18 +785,18 @@ public function getFirstChild($uid)
      *
      * @return int
      */
-    public function getLastChild($uid)
+    public function getLastChild(int $uid): int
     {
         $connectionPool = GeneralUtility::makeInstance(ConnectionPool::class);
         $queryBuilder = $connectionPool->getQueryBuilderForTable('tx_dlf_documents');
 
         $child = $queryBuilder
             ->select(
-                'tx_dlf_documents.uid AS uid'
+                'uid'
             )
             ->from('tx_dlf_documents')
             ->where(
-                $queryBuilder->expr()->eq('tx_dlf_documents.partof', (int) $uid)
+                $queryBuilder->expr()->eq('partof', (int) $uid)
             )
             ->add('orderBy', 'volume_sorting desc')
             ->addOrderBy('tx_dlf_documents.volume_sorting')
```

Comment:
```suggestion
            ->add('orderBy', 'volume_sorting desc')
            ->addOrderBy('volume_sorting')
```
As a side note: Aren't those two statements redundant? (Which applies to almost all methods of this class...)

Reply:
I was also thinking about it. Just the first one contains `desc` an second one not. I will check it out.